aten/src/ATen/mps/MPSAllocator.mm

```diff
@@ -141,6 +144,14 @@
   return it->second;
 }
 
+bool MPSHeapAllocatorImpl::trigger_memory_callbacks(BufferBlock* buffer_block, IMpsAllocatorCallback::EventType event) {
+    bool result = false;
```

Comment:
What is this boolean for? We are passing this boolean all the way down the chain, but doesn't seem to be getting used for anything

Reply:
If this is not used, can we please remove it.

Reply:
Th code was imported from Cuda's CachingAllocator. This is a generic callback which could also be used when we do allocate or recycle buffers. In the future we may see a case where we need to return results from callback to MPSAllocator. Anyway, I'll remove it for now.